website/content/docs/v0.14/Learn More/architecture.md

```diff
@@ -18,6 +18,17 @@ This imposes a clear separation of concerns and ensures that changes over time w
 The benefit is that each component may be iterated and changed as its needs dictate, so long as the external API is controlled.
 This is a key component in reducing coupling and maintaining modularity.
 
+## File system partitions
+
+Talos uses these partions with the following labels:
+
+    - **EFI**: The EFI partition, which is the first partition on the disk and used for EFI boot.
+    - **BIOS**: The BIOS partition, which is the second partition on the disk and used for GRUB's second stage boot.
+    - **BOOT**: The boot partition, which is the third partition on the disk and used for the bootloader.
+    - **META**: The meta partition, which is the fourth partition on the disk and used for storing metadata about the talos node, such as node id's.
+    - **STATE**: The state partition, which is the fifth partition on the disk and used for storing persistent state information.
```

Comment:
I would probably clarify here that it stores machine configuration and some node identity data for cluster discovery and KubeSpan